lib/pbench/server/api/resources/query_apis/month_indices.py

```diff
@@ -1,92 +1,43 @@
 from logging import Logger
 
 from flask import jsonify
-from flask_restful import Resource, abort
-import requests
+from requests.models import Response
+from typing import Any, AnyStr, Dict
 
 from pbench.server import PbenchServerConfig
-from pbench.server.api.resources.query_apis import get_es_url, get_index_prefix
+from pbench.server.api.resources.query_apis import ElasticBase
 
 
-class MonthIndices(Resource):
+class MonthIndices(ElasticBase):
     """
     Get the range of dates in which datasets exist.
     """
 
     def __init__(self, config: PbenchServerConfig, logger: Logger):
-        """
-        __init__ Initialize the resource with info each call will need.
-
-        Args:
-            :config: The Pbench server config object
-            :logger: a logger
-        """
-        self.logger = logger
-        self.es_url = get_es_url(config)
-        self.prefix = get_index_prefix(config)
+        super().__init__(
+            config, logger, {},
+        )
```

Comment:
I suspect, if you remove the trailing comma from line 18, that Black will format this all on a single line, which would be nicer.

Reply:
Perhaps I'm being pedantic, but shouldn't you be passing `Schema()` (or `Schema([])`) instead of `{}` as the third parameter?